react-ui/src/components/pages/SignUp.js

```diff
@@ -98,14 +102,16 @@ class SignUp extends React.Component {
     handleSubmit = (e) => {
         e.preventDefault();
         this.props.userSignUp(this.state)
+        this.setState({ loading: true })
```

Comment:
I'm kind of confused why we are using a component-level state.loading here. Doesn't this defeat the purpose of redux?